packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.scss

```diff
@@ -7,17 +7,62 @@
 // --------------------------------------------------
 // Search box styles
 
-
-$SearchBox-field-padding-left: 10px;
-$SearchBox-height: 36px;
-
 .root {
   @include ms-font-m;
   @include ms-u-normalize;
   color: $ms-color-neutralPrimary;
   position: relative;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: stretch;
   margin-bottom: 10px;
```

Comment:
that was already there. I'd certainly love to get rid of it, but out of scope of this update.